inc/gui.class.php

```diff
@@ -179,7 +179,7 @@ public static function options_page() { ?>
 								<input type="checkbox" name="ab_gravatar_check" id="ab_gravatar_check" value="1" <?php checked($options['gravatar_check'], 1) ?> />
 								<label for="ab_gravatar_check">
 									<?php esc_html_e( 'Trust commenters with a Gravatar', 'antispam-bee' ); ?>
-									<span><?php esc_html_e( 'Check if commenter has a Gravatar image', 'antispam-bee' ); ?></span>
+									<span><?php _e( 'Check if commenter has a Gravatar image. Please note the <a href="https://github.com/pluginkollektiv/antispam-bee/wiki/en-Documentation#trust-commenters-with-a-gravatar" target="_blank" rel="noopener noreferrer">privacy notice</a> for this option.', 'antispam-bee' ); ?></span>
```

Comment:
That's a good point. Not quite sure, how about:
```
$tag = sprintf(
    '<a href="%s">',
    esc_url( __( 'http://...', 'antispam-bee' ) )
);
sprintf(
	esc_html__( 'Check ... %sprivacynotice%s ...' ), $tag,
	'</a>'
);
```
Again maybe with a translation notice.